prov/verbs/src/windows/verbs_nd_ov.c

```diff
@@ -313,29 +325,48 @@ void nd_cm_connect_ack(struct nd_event_base *base, DWORD bytes)
 		if (!cm_event)
 			cm_event = nd_allocate_cm_event(
 				id, RDMA_CM_EVENT_ESTABLISHED);
+		break;
```

Comment:
Th first part of this comment applies to code 8 lines up, only I can't comment there.  The comment above says "// Intentional fall through."  I know some compilers will recognize "/* fall through */" as a comment to avoid generating a warning about a missing break statement.  I don't know what the windows compiler recognizes for this.

The second part of this comment is why did we fall through?  The comment under case ND_CM_ACCEPT looks like it's backwards.  We have an cm_event if we fell through.  And in falling through, we don't actually do anything but break.